shepherd_bms/src/analyzer.cpp

```diff
@@ -384,4 +384,18 @@ void Analyzer::disableTherms()
             }
         }
     }
+}
+
+void Analyzer::calcPackResistance(node *bms_data)
+{
+    if(bms_data->data.packCurrent >= 100)
```

Comment:
The way this is implemented we are effectively going to get the resistance calculated when the current spike is ending. Unsure if this is what we want. Fine for now, but we should keep in mind. Also should consider that we are wasting resources recalculating this constantly when only the value right at 100-105 is gonna be used.